test/smoke/loop-dir-2/Makefile

```diff
@@ -14,5 +16,3 @@ CC           = $(OMP_BIN) $(VERBOSE)
 #"-\#\#\#"
 
 include ../Makefile.rules
-# run: $(TESTNAME)
-# 	$(RUNENV) ./$(TESTNAME) 2>&1 | $(FILECHECK) $(TESTSRC_MAIN)
```

Comment:
RUNENV here is present in the custom rule but it is commented out above. Is this intentional?

Reply:
Yes, intentional. The test had RUNENV commented out for some reason. Perhaps because no CHECK line in the source.

Reply:
Sounds good!